src/azure-data-transfer/azext_azure_data_transfer/_help.py

```diff
@@ -0,0 +1,61 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['azure-data-transfer pipeline approve-connection'] = """
+type: command
+short-summary: Approves the specified connection in a pipeline.
```

Comment:
Is this summary manually added? I think we [updated ](https://github.com/Azure/azure-rest-api-specs/blob/2e9eecb85170aec309ca95949428c0ae3cee0b78/specification/azuredatatransfer/resource-manager/Microsoft.AzureDataTransfer/stable/2025-05-21/azuredatatransfer.json#L2314C1-L2315C1)this to ` Approves the specified connection request in a pipeline.`

Can we check for other cmdlets too to maintain consistency?